### ISSUE TITLE
UseCustomReportImagePath not working

### ISSUE THREAD

**Issue by alicia90** [alanjuden/MvcReportViewer#84]
I override the property "UseCustomReportImagePath" to true:

` protected override bool UseCustomReportImagePath
        {
            get
            {
                return true;
            }
        }`

And the property of 

```
protected override string ReportImagePath
        {
            get
            {
                return "/Report/ReportImage/?originalPath={0}";
            }
        }
```
But the images in my report no found. The URL generated is : 
http://ReportServer&rs%3ASessionID=zf1gau45cjcc0k45o2kus245&rs%3AFormat=HTML4.0&rs%3AImageID=I_1409_1_H

Becouse?

**Comment by alicia90** [comment]
The image is stored in a BBDD as field type image

**Comment by alanjuden** [comment]
@alicia90, You've overridden the ReportImagePath to the same value that is defined in the ReportController I created...which has really done nothing for you. If you're hoping to use this on a report server that is internally available you really want to override the "UseCustomReportImagePath" property and set the return value to true. Currently, your images are using the path that is coming from the Report Server. If you set UseCustomReportImagePath = true, then the website will begin acting as a proxy to grab the images for you.

Alan

**Comment by alicia90** [comment]
Yes, I set UseCustomReportImgePath = true. But the website don't begin acting as a proxy to grab the images.
